src/plugin/svipc/sysvipcwrappers.cpp

```diff
@@ -108,14 +122,22 @@ void *shmat(int shmid, const void *shmaddr, int shmflg)
 }
 
 extern "C"
+bool dmtcp_svipc_inside_shmdt()
```

Comment:
I'd say let's use EXTERNC. This will allow us to use this function from a C file as well.